async-download-rest/src/test/java/org/uniprot/api/async/download/controller/UniRefDownloadControllerIT.java

```diff
@@ -306,4 +315,29 @@ protected String getResultIdStringToMatch() {
     protected DownloadJobRepository getDownloadJobRepository() {
         return this.downloadJobRepository;
     }
+
+    @Override
+    protected String getIdsFolder() {
```

Comment:
I think here we could use the new Async config that you created. it would replace many methods to just one